chord_metadata_service/phenopackets/tests/constants.py

```diff
@@ -498,16 +498,16 @@ def valid_phenotypic_feature(biosample=None, phenopacket=None):
         },
         excluded=True,
         severity={
-            "id": "HP: 0012825",
+            "id": "HP:0012825",
             "label": "Mild"
         },
         modifiers=[
             {
-                "id": "HP: 0012825 ",
+                "id": "HP:0012825",
                 "label": "Mild"
             },
             {
-                "id": "HP: 0012823 ",
+                "id": "HP:0012823",
                 "label": "Semi-mild"
```

Comment:
probably not relevant to the test, but `HP:0012823` is the ontology for a modifier, rather than a modifier itself. I'm guessing this is meant to be `HP:0012826` - "Moderate" or something similar.